src/main/java/net/greatstart/dao/hibernate/EventDaoImpl.java

```diff
@@ -2,29 +2,34 @@
 
 import net.greatstart.dao.EventDao;
 import net.greatstart.model.Event;
+import org.springframework.stereotype.Repository;
 
 import java.util.List;
 
-
+@Repository
 public class EventDaoImpl implements EventDao {
 
-    public boolean create(Event item) {
+    @Override
+    public boolean create(Event event) {
         return false;
     }
 
-    public boolean delete(long id) {
+    @Override
+    public boolean delete(Event event) {
         return false;
     }
 
-    public boolean update(Event item) {
+    @Override
+    public boolean update(Event event) {
         return false;
     }
 
-
+    @Override
     public Event getById(long id) {
         return null;
     }
 
+    @Override
     public List<Event> getAll() {
         return null;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1168)